curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-describe-places-and-events/66a909dc2e382527ead44c23.md

```diff
@@ -4,9 +4,10 @@ title: Task 1
 challengeType: 19
 dashedName: task-1
 ---
+
 <!-- (Audio) Bob: Hey Lisa, do you have a minute?  -->
 
-# --instructions--
+# --description--
```

Comment:
weren't instructions added in a PR recently?

Reply:
https://github.com/freeCodeCamp/freeCodeCamp/pull/56472/files it is intended, it is the first application of the new instructions section

Reply:
cc/ @moT01

Reply:
We can probably just make `description` default to an empty string and delete the warning. We already pretend that the type of `description` is `string`, after all, so all the client code is written under the assumption that it is defined.